docs/en/admins/10_ServerConfig.md

```diff
@@ -91,7 +91,10 @@ server {
 		fastcgi_split_path_info ^(.+\.php)(/.*)$;
 		# By default, the variable PATH_INFO is not set under PHP-FPM
 		# But FreshRSS API greader.php need it. If you have a “Bad Request” error, double check this var!
-		fastcgi_param PATH_INFO $fastcgi_path_info;
+		# NOTE: the separate $path_info variable is required. For more details, see:
```

Comment:
If I understand correctly, this nginx bug is only when using `try_files`, right?
https://forum.nginx.org/read.php?2,238825,239198#msg-239198

Reply:
Yes, it works fine without `try_files`, however I'd recommend including the advisory, as `try_files` directives are quite common, and it's possible other users might make changes to the example config, and have issues with their API access.